src/node/blockstorage.h

```diff
@@ -236,11 +240,11 @@ class BlockManager
     bool ReadBlockFromDisk(CBlock& block, const FlatFilePos& pos, const Consensus::Params& consensusParams) const;
     bool ReadBlockFromDisk(CBlock& block, const CBlockIndex* pindex, const Consensus::Params& consensusParams) const;
```

Comment:
Thank you for the suggestion, will split this up.

Reply:
In commit "refactor: Move functions to BlockManager methods" (1def580e80a1892e473324016db9a0f2ffec0c27)

re: https://github.com/bitcoin/bitcoin/pull/27125#discussion_r1140490220

> Thank you for the suggestion, will split this up.

Seems like this could be marked resolved